src/App.js

```diff
@@ -1,11 +1,11 @@
 import './App.css';
 
-import Greetings from './components/greetings';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] As it is a requirement for this project, all your components are supposed to `class-based`, checkout the [seventh requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) and kindly fix that.

Reply:
Hello @Mihndim2020 Thank you for reviewing my project.
I believe I did use class-based components. Kindly have a look at my calculator component [here](https://github.com/Nemwel-Boniface/math-magicians/blob/calculatorUI/src/components/Calculator.js).
Do correct me if I am wrong.

Reply:
Hello @Mihndim2020  I made the required changes.